<?xml version="1.0" ?>
<launch>
	<rosparam file="$(find butia_speech)/config/ros.yaml" command="load"/>
	
	<node name="speech_recognizer" pkg="butia_speech" type="speech_recognizer.py" output="screen"/>
</launch>
